content/en/docs/ops/ambient/getting-started/index.md

```diff
@@ -6,8 +6,21 @@ weight: 1
 owner: istio/wg-networking-maintainers
 test: n/a
 ---
+
 {{< warning >}}
-Ambient is currently in alpha status, not intended for production use.
+Ambient is currently in [alpha status](/docs/releases/feature-stages/#feature-phase-definitions).
+
+Please thoroughly review the [feature phase definitions](/docs/releases/feature-stages/#feature-phase-definitions) before use.
+
+In the past, smaller "alpha" features have often been introduced without any issues, and later promoted to "stable" without changes.
+This may give users an over-optimistic view of the stability and safety guarantees of an "alpha" feature.
+
+Given the scope of ambient mode, it is imperative to treat the feature as dictated by the feature phase definitions.
+In particular, there are already known performance, stability, and security issues in the alpha release.
+There are also planned breaking changes, including those that will prevent upgrades.
+These are all limitations that will be addressed before graduation to "beta".
+
```

Comment:
Do we really need to highlight these? per alpha definition which you linked, it is targeted to developers and expert for early feedback, and no guarantees on backward compatibility, may have obvious security vulnerabilities etc.

Reply:
Given ~all users run alpha features in prod I doubt it is enough without this.